gogs/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,12 @@
+---
+openshift_cluster_content:
+- object: projectrequest
+  content:
+  - name: gogs-project
+    file: "{{ inventory_dir }}/../files/projects/projects.yml"
```

Comment:
If the applier is run as a regular user you'll also need to add `file_action: create`
Otherwise can you add a section to the README detailing what type of permissions are required?

Reply:
I have added a note in the instructions to specify that this requires a privileged user as `anyuid` SCC is modified.